tests/notebooks/linear_decoder.ipynb

```diff
@@ -303,6 +246,7 @@
     }
    ],
    "source": [
+    "n_epochs = 50 if n_epochs_all is None else n_epochs_all\n",
     "trainer.train(n_epochs=50)"
```

Comment:
should this be n_epochs=n_epochs ?

Reply:
It should't be, this allows n_epochs to be set to 1 in test mode. Our testing code finds n_epochs_all and makes it 1.